apple/swift-docc-render
# Remove duplicate languagePaths and fix issue with variants being undefined

Bug/issue #, if applicable: 89691775

## Summary

This fixes potential issues when there are no language `variants` in the RenderJSON for a technology.

## Dependencies

NA

## Testing

[SlothCreator 2.doccarchive.zip](https://github.com/apple/swift-docc-render/files/8172001/SlothCreator.2.doccarchive.zip)


Steps:
1. Use the attached archive
2. Go to http://localhost:8080/documentation/slothcreator
3. Make sure it does not error out anymore

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
